first_problem_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test/unit'
+require_relative'./first_problem'
+
+class FirstProblemTest < Test::Unit::TestCase
+
+  # Called before every test method runs. Can be used
+  # to set up fixture information.
+  def setup
+    # Do nothing
+  end
+
+  # Called after every test method runs. Can be used to tear
+  # down fixture information.
+
+  def teardown
+    # Do nothing
+  end
+
+  def test_compute_emptyArray_0
+    array = Array.new
+    assert_equal(0,FirstProblem.new(array).compute)
+  end
+
+  def test_compute_noEvenNumbersArray_0
+    array = [1]
+    assert_equal(0,FirstProblem.new(array).compute)
+
+    array = [1,3,5,7]
+    assert_equal(0,FirstProblem.new(array).compute)
+  end
+
+  def test_compute_validNumbersArray_correctResult
+    array = [1,3,8]
+    assert_equal(8,FirstProblem.new(array).compute)
```

Comment:
You should have a separate unit test for each of the asserts here

Reply:
I would argue the same as before , since we respect the expected input , there should be no need for multiple unit tests